english/candyland/diabetes.md

```diff
@@ -0,0 +1,19 @@
+You light a small fire in the middle of the marshmallow room.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.